README.md

```diff
@@ -154,27 +154,6 @@ The documentation is available here: [documentation](https://pypsa-earth.readthe
 <!-- readme: collaborators,contributors -start -->
 <table>
 <tr>
-    <td align="center">
-        <a href="https://github.com/hazemakhalek">
-            <img src="https://avatars.githubusercontent.com/u/26235356?v=4" width="100;" alt="hazemakhalek"/>
-            <br />
-            <sub><b>Null</b></sub>
-        </a>
-    </td>
-    <td align="center">
-        <a href="https://github.com/jarry7">
-            <img src="https://avatars.githubusercontent.com/u/27745389?v=4" width="100;" alt="jarry7"/>
-            <br />
-            <sub><b>Jarrad Wright</b></sub>
-        </a>
-    </td>
-    <td align="center">
-        <a href="https://github.com/fneum">
-            <img src="https://avatars.githubusercontent.com/u/29101152?v=4" width="100;" alt="fneum"/>
-            <br />
-            <sub><b>Fabian Neumann</b></sub>
-        </a>
-    </td>
```

Comment:
This stuff re-appeared. I feel like your local fork needs cleaning.
When you do git merge main, you generally merge the local version of your github.
If your local main is not synced with the upstream main (hence your github fork not synced with the upstream and the local fork not synced with your github fork).